addons/web/static/src/js/chrome/keyboard_navigation_mixin.js

```diff
@@ -44,6 +45,21 @@ odoo.define('web.KeyboardNavigationMixin', function (require) {
             this._areAccessKeyVisible = false;
             this.BrowserDetection = new BrowserDetection();
         },
+        /**
+         * @override
+         */
+        start: function () {
+            this._hideAccessKeyOverlay = this._hideAccessKeyOverlay.bind(this);
+            window.addEventListener('blur', this._hideAccessKeyOverlay);
```

Comment:
The event argument was masked before, seems better to keep doing it. Probably just:
```suggestion
            this._hideAccessKeyOverlay = () => this._hideAccessKeyOverlay();
            window.addEventListener('blur', this._hideAccessKeyOverlay);
```